Website/Providers/DataProviders/SqlDataProvider/09.02.02.SqlDataProvider

```diff
@@ -0,0 +1,81 @@
+/************************************************************/
+/*****              SqlDataProvider                     *****/
+/*****                                                  *****/
+/*****                                                  *****/
+/***** Note: To manually execute this script you must   *****/
+/*****       perform a search and replace operation     *****/
+/*****       for {databaseOwner} and {objectQualifier}  *****/
+/*****                                                  *****/
+/************************************************************/
+
+/* Update version number in DesktopModules */
```

Comment:
Do we have a history as to WHY we are doing this?  The installation of the various module packages update the version to the version that is referenced in the install package.  This is essentially duplicating the work of the installer.

Reply:
@zyhfish @ashishpd, any insights?

Reply:
I do know that we have to have a 09.02.02.sqldataprovider file as things get weird without it.  I'm curious on the actual content/process within

Reply:
each release need a sqldataprovider file for it, even it may contains nothing. AFAIK currently we nee update legacy module's version in each upgrade.

Reply:
Are these extensions that are getting included automatically and aren't part of any extension package?

Reply:
these are the core data providers, doesn't belong to any extension package.

Reply:
I don't see a need to update version number of components, which haven't been modified.
Of course, they can't be applied to previous versions, but there is no need to update its registration.

But this has been done like this since DNN 7.0 AFAIR, IMO we shouldn't change before 10.0.1